### DISCUSSION TITLE
Import issue after deleting account

### DISCUSSION THREAD

**Discussion by anantone** [firefly-iii/firefly-iii#8652] [General]
Hi,

During my initial import, transactions were linked to the wrong account. I subsequently deleted that account and transferred the transactions to the correct one. Now when running the installer, all these transactions are marked as errors with `Import index 300: transactions.0.source_id: This value is linked to an object that does not seem to exist. (original value: "1")`

 Is there a way that I can reset this, to avoid getting hundreds of errors and an email with each daily import? 

Thanks! (posting here as not a bug, hope that's right)
Antoine

**Comment by JC5** [comment]
So you're running the importer again, right? How exactly?

**Reply by anantone** [reply]
Through docker exec:
`docker exec firefly_iii_importer php artisan importer:import /import/config.json `

**Reply by anantone** [reply]
I had done the initial import through the web UI, then docker exec with the config.json from that.

**Reply by JC5** [reply]
In that case, update the JSON file. There is a reference to default account ID. Swap "1" for the new one.

**Reply by anantone** [reply]
Thanks! But the default account ID is already on "1". Is there somewhere to see which account has which ID?

**Reply by JC5** [reply]
That's why, you must change it. Check the URL of your new account.

**Reply by anantone** [reply]
Ah, got it. So in the URL https://example.com/accounts/show/x the account ID is x. 
Yet I've changed that in the config.json, recreated the importer container, and am getting the same error when trying to import.

**Reply by JC5** [reply]
Did you map accounts during the configuration?

**Reply by anantone** [reply]
Turns out I also had to update the account ID lower in the config.json file, where the accounts are listed. Problem solved. Thank you for helping so quickly!

**Reply by JC5** [reply]
Ah, nice, sorry I didn't think about that list as well. Glad to hear it works!
